packages/core/src/interpreter.ts

```diff
@@ -486,6 +490,13 @@ export class Interpreter<
       return;
     }
 
+    if (((event as EventObject)?.type || '').includes('error')) {
+      if (this.parent) {
+        this.parent.send(event);
+        return;
+      }
+    }
```

Comment:
how this relates to the logic introduced within `handleErrorEvent`? It feels like we'd never get to that other logic if we return from here. Is the distinction that in here we might handle "non-scxml" error event?

Reply:
you are right, the logic introduced within handleErrorEvent is not needed with this change